app/models/picture.rb

```diff
@@ -0,0 +1,6 @@
+class Picture < ActiveRecord::Base
+attr_accessor
+  has_attached_file :image, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: ":style/monkey.png"
```

Comment:
Metrics/LineLength: Line is too long. [111/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)